lib/rufus/lua/lib_lua_5_3.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Rufus::Lua
+  module Lib
+    module Lua5_3
+      def self.included(mod)
+        mod.attach_function :lua_callk, [ :pointer, :int, :int, :int, :int ], :void
+
+        mod.attach_function :lua_pcallk, [ :pointer, :int, :int, :int, :int, :int ], :int
+
+        mod.attach_function :lua_tonumberx, [ :pointer, :int, :pointer ], :double
+
+        mod.attach_function :lua_rawlen, [ :pointer, :int ], :int
+
+        mod.attach_function :lua_rotate, [ :pointer, :int, :int ], :void
+
+        mod.attach_function :luaL_loadbufferx, [ :pointer, :string, :int, :string, :pointer ], :int
+
+        mod.attach_function :lua_rawlen, [ :pointer, :int ], :int
+
+        mod.attach_function :lua_setglobal, [ :pointer, :string ], :void
+        mod.attach_function :lua_getglobal, [ :pointer, :string ], :int
+        mod.attach_function :luaL_requiref, [ :pointer, :string, :cfunction, :int ], :void
+
+        mod.extend(ClassMethods)
+      end
+
+      module ClassMethods
+        def lua_call(pointer, nargs, nresults)
+          lua_callk(pointer, nargs, nresults, 0, 0)
+        end
+
+        def lua_pcall(pointer, nargs, nresults, msgh)
+          lua_pcallk(pointer, nargs, nresults, msgh, 0, 0)
+        end
+
+        def lua_tonumber(pointer, index)
+          lua_tonumberx(pointer, index, FFI::Pointer::NULL)
+        end
+
+        def lua_objlen(pointer, index)
```

Comment:
Public method was renamed to `lua_rawlen`:
https://www.lua.org/source/5.3/luaconf.h.html#lua_objlen